### DISCUSSION TITLE
Older format of config parser - can't load the database

### DISCUSSION THREAD

**Discussion by Gentleman03** [RafaelGB/obsidian-db-folder#711] [General]
### Contact Details

gentleman3job@gmail.com

### What happened?

When i try to load my folder db, file is blank and in console there is an error

### What platform were you using?

Desktop

### Version of the plugin

3.1.2

### Relevant log output

```shell
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'columns') plugin:dbfolder:118 
    at AQ.eval (plugin:dbfolder:118:58040)
    at Generator.next (<anonymous>)
    at o (plugin:dbfolder:1:748)
```

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
could you share the content of your ddbb markdown file?

Maybe something its bad parsed internally

**Comment by Gentleman03** [comment]
> could you share the content of your ddbb markdown file?
> 
> Maybe something its bad parsed internally

https://pastebin.com/9aXd1fKQ
password: 333

**Comment by RafaelGB** [comment]
Looks like an old format (3-4 months ago) and with 3.0.0 was deprecated

substituting `%% dbfolder:yaml` with ```yaml:dbfolder

And the end of the config %% with ```

you can always create an empty ddbb and check the diffs

**Comment by Gentleman03** [comment]
Indeed, that solved the issue, thank you :)
